inventories/group_vars/all/manifest.yaml

```
@@ -7,7 +7,7 @@ integreatly_version: master
 threescale: True
 # below threescale variables below are not currently used. They define where and what to pull in from github. These will be important as we look to break out and decouple the resources from the installer ###############
 # not currently used but records the version installed by the operator, this should come from the status block
-threescale_version: '2.9.0.GA'
+threescale_version: '2.10.0.GA'
```

```suggestion
threescale_version: '3scale-2.10.0-GA'
```
Apparently 3scale changed their tagging convention to include a 3scale- prefix starting with 2.10.0.  without this, install of 2.10 on fresh cluster won't work.